CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to Moody
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to Moody. These are mostly guidelines, not rules. 
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+Please treat everybody with respect. If you do not get reply on your questions/issues, please be patient.
+Author or maintainers will reply as soon as possible.
+
+## What should I know before I get started?
+
+You should have an understanding of developing chrome extensions and knowledge about Git.
+
+
+
+## How Can I Contribute?
+
+1. Find an issue that you are interested in addressing or a feature that you would like to add.
```

Comment:
@mann-codes2014 Add a point here to create a new issue and get assigned before working on it.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Yes sure